src-js/views-fontinfo/src/panel-axes.js

```diff
@@ -917,7 +917,7 @@ export function updateRemoveButton(list, buttons) {
   });
 }
 
-function arraysEqual(arrayA, arrayB) {
+export function arraysEqual(arrayA, arrayB) {
```

Comment:
I don't think you actually import this function, so this change isn't needed.

Reply:
Thanks. Done.